perses/analysis/visualization.py

```diff
@@ -0,0 +1,404 @@
+        self._mutated_residue = mutated_residue
+        self._as_spheres = as_spheres
+        self._ligand_chain = ligand_chain
+
+        # Set selection strings for mutated residue/ligand
+        if self._is_protein:
+            if self._mutated_residue is None:
+                raise Exception("Need to specify the mutated residue id!")
+            self._old_selection = f"(old and chain {self._mutated_chain} and resi {self._mutated_residue})"
+            self._new_selection = f"(new and chain {self._mutated_chain} and resi {self._mutated_residue})"
+            self._both_selection = f"(chain {self._mutated_chain} and resi {self._mutated_residue})"
+        else:
+            self._old_selection = f"(old and chain {self._mutated_chain})"
+            self._new_selection = f"(new and chain {self._mutated_chain})"
+            self._both_selection = f"chain {self._mutated_chain}"
+
+        # Get PyMOL indices of unique old and new atoms as strings
+        _logger.info("Getting unique old and new atom indices...")
+        self._unique_old = f"(old and chain {self._mutated_chain} and resi {self._mutated_residue} and not backbone)"
+        self._unique_new = f"(new and chain {self._mutated_chain} and resi {self._mutated_residue} and not backbone)"
+
+    def load(self):
+        """
+        Load the perses trajectory into PyMOL.
+
+        """
+        _logger.info("Loading trajectory...")
+
+        # Launch pymol session
+        pymol.pymol_argv = ["pymol", "-qc"] + sys.argv[1:]
+        pymol.finish_launching()
+
+        # Load trajectories
+        cmd.set("defer_builds_mode", 3)  # Improve performance for handling long trajectories
+        if os.path.exists(self._old_pdb):
+            cmd.load(self._old_pdb)
+        else:
+            raise FileNotFoundError(f"{self._old_pdb} not found")
+        if os.path.exists(self._old_traj):
+            cmd.load_traj(self._old_traj, state=1)
+        else:
+            raise FileNotFoundError(f"{self._old_traj} not found")
+        if os.path.exists(self._new_pdb):
+            cmd.load(self._new_pdb)
+        else:
+            raise FileNotFoundError(f"{self._new_pdb} not found")
+        if os.path.exists(self._new_traj):
+            cmd.load_traj(self._new_traj, state=1)
+        else:
+            raise FileNotFoundError(f"{self._new_traj} not found")
+
+        # Rename objects
+        old_name = os.path.splitext(os.path.basename(self._old_pdb))[0]
+        new_name = os.path.splitext(os.path.basename(self._new_pdb))[0]
+        cmd.set_name(old_name, "old")
+        cmd.set_name(new_name, "new")
+
+        # Check that the trajectories are the same length
+        old_states = cmd.count_states("old")
+        new_states = cmd.count_states("new")
+        if old_states != new_states:
+            raise Exception(f"The old and new trajectories are not the same length! old: {old_states}, new: {new_states}")
+
+        # Remove solvent
+        _logger.info("Removing solvent...")
```

Comment:
maybe keep solvent near binding site? idk

Reply:
Keeping solvent near the binding site isn't super useful as it changes across frames, so watching solvent in a movie just look like a lot of bouncing around